packages/terra-graphs-docs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cerner/terra-graphs-docs",
-  "version": "0.0.0",
+  "version": "0.0.1",
   "private": true,
```

Comment:
The `private` field should be deleted in order to release it.

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/274/commits/81caeac55d11d0c6e5a2d9d6a8a544a65aaa6c2f#diff-d761c3fb8ba076a072500ff322d01b361058688b30dff1e2cd8fd703d36b6da2L4).